Software/Swoffsets.md

```diff
@@ -50,7 +51,15 @@ Whichever side you choose, you need to keep it consistent across *both robots (c
 1. Redeploy code to the robot, now with the correct offsets
 2. Open AdvantageScope and navigate to the `Swerve` tab
 3. Drag over the `Desired States` and `Actual States` to `Sources` on the bottom of the window 
-4. Open Driverstation, connect a controller, and begin driving
+4. Open Driverstation, connect a controller, reset your pose to be relative to you, then begin driving
 5. If pressing up on the joystick makes the robot drive forwards, you've done swoffsets correctly! 🎉
 6. Verify that the `Desired States` mostly match the `Actual States` using the AdvantageScope UI. Revel in how cool they look. Then go get some fruit snacks because you've earned it 😎
 ![AdvantageScope](../.images/Software/Swoffsets/AdvantageScope_(WPILib)_HAoyjhRLd6.png)
+
+## Common Failures:
```

Comment:
One more failure mode we should add is:
- steer bearing is broken, if the swofsets are fine on boot but drift over time (most commonly just one module) this indicates it is not a magnet issue and instead the steering module bearing or belt is broken